rest_api/config_apis/imagedigestmirrorset-config-openshift-io-v1.adoc

```diff
@@ -115,7 +115,10 @@ Required::
 
 | `mirrorSourcePolicy`
 | `string`
-| mirrorSourcePolicy defines the fallback policy if fails to pull image from the mirrors. If unset, the image will continue to be pulled from the the repository in the pull spec. sourcePolicy is valid configuration only when one or more mirrors are in the mirror list.
+| `mirrorSourcePolicy` defines the fallback policy if fails to pull image from the mirrors. If unset, the image will continue to be pulled from the the repository in the pull spec. `sourcePolicy` is valid configuration only when one or more mirrors are in the mirror list.
```

Comment:
Even though this was an issue before this PR, I think this first sentence is missing a word. It should specify what specify what fails to pull images:
```suggestion
| `mirrorSourcePolicy` defines the fallback policy if {something} fails to pull image from the mirrors. If unset, the image will continue to be pulled from the the repository in the pull spec. `sourcePolicy` is valid configuration only when one or more mirrors are in the mirror list.
```

Up to you to fix it since this is out of scope to your work, but I think it would be a valuable thing to clarify.

Reply:
How about- " `mirrorSourcePolicy` specifies the fallback policy to use if pulling an image from the mirrors fails. If this policy is not set, the image will continue to be retrieved from the repository specified in the pull spec. The `sourcePolicy` configuration is valid only when one or more mirrors are included in the mirror list."?

Reply:
Looks great! Keep in mind that this PR's update also needs to be applied against the repo where this API reference is pulled from